src/scope/FunctionScope.js

```diff
@@ -37,18 +36,18 @@ class FunctionScope extends Scope {
     if (this.parent) {
       return this.parent.findFunctionScoped(name);
     }
-    return false;
+    return [];
   }
 
   /**
-   * Looks up variable from block scope.
+   * Looks up variables from block scope.
    * (i.e. the parent block scope of the function scope.)
    *
    * When variable found from function scope instead,
    * returns false to signify it's not properly block-scoped.
    *
    * @param  {String} name Variable name
-   * @return {Variable} The found variable or false
+   * @return {Variable[]} The found variables (empty array if none found)
    */
   findBlockScoped(name) {
     if (this.vars[name]) {
```

Comment:
So, we're now returning `false` in here but the calling code always expects an array.

But for none of the tests fails. This means we're missing a test for the scenario where this condition in `FunctionScope.findBlockScoped()` is true.